uctMaths/competition/interface/school_report.html

```diff
@@ -44,58 +44,110 @@ <h2>School report for <u>{{schooln}}</u></h2>
+                            <table> 
+                            <tr>
+                                <th align="left">Reference </th>
+                                <th align="left">First Name</th>
+                                <th align="left">Surname</th>
+                                <th width="40" align="left">Score</th>
+                                <th width="30" align="left">Rank</th>
+                                <th width="30" align="left"></th>
+                            </tr>
+                            <!-- Populates the list with previously registered students -->
+                            {%for key, student_l in student_list.items%}
+                                {% if grade == key %} <!-- only for the current grade -->
+                                    {%for student in student_l%}
+                                        <tr>
+                                        <td>{{student.reference}}</td>
+                                        <td>{{student.firstname}}</td>
+                                        <td>{{student.surname}}</td>
+                                        <td>{%if student.score > 0%}
+                                            {{student.score}}
+                                            {%else%}
+                                            ABSENT
+                                            {%endif%}</td>
+                                        <td>{%if student.award%}
+                                            {{student.rank}}
+                                            {%endif%}</td>
+                                        <td>{%if 'G' == student.award%}
+                                            Gold
+                                            {%elif 'M' == student.award%}
+                                            Merit
+                                            {%elif 'MOX' == student.award%}
+                                            Merit, School Award
+                                            {%elif 'OX' == student.award %}
+                                            School Award
+                                            {%endif%}
+                                            </td>
+                                        </tr>
+                                    {%endfor%}
+                                {%endif%}
                             {%endfor%}
-                        {%endif%}
-                    {%endfor%}
-                    </table>
-                </div>
-        {%endfor%}
+                            </table>
+                        {%endfor%}
+                        </td>
+                    </tr>
+                </table>
+
         
     <div>
     <br><br>
-    <b>{{school_award_blurb}}</b>
+    <b>{{school_award_blurb|linebreaks}}</b>
     <br><br>
-    The UCT Mathematics Competition team thanks you for your school's participation in the annual UCT Mathematics Competition.
```

Comment:
Was this removed intentionally?

Reply:
Yes, Prof Web wanted the summary at the end to change completely. 
![image](https://user-images.githubusercontent.com/96055906/150107400-58f6be18-e96f-4df8-8243-84c38b3a41d2.png)